sdk/appconfiguration/ci.yml

```diff
@@ -21,6 +21,7 @@ pr:
     include:
     - sdk/appconfiguration/
     - sdk/resourcemanager/
+    - sdk/network/
```

Comment:
We can remove this after we GA network and change the reference in tests to PackageReference.